Server/All

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/python
+
+# Turn on debug mode.
+import cgitb
+import pymysql
+import json
+
+cgitb.enable()
+
+# Print necessary headers.
+print("Content-Type: text/html; charset=utf-8\n\n")
+
+import cgi
+form = cgi.FieldStorage()
+print ("Hello")
+print ("<br />")
+print ("<br />")
+
+#def greet_player():
+    #if "player" not in form:
+        #print ("Something Went Wrong")
+    #else:
+        #print("<b>Hello " + str(form.getvalue("player")) + "</b>")
+        #my_first_int = 1
+        #my_first_str = str(form.getvalue("user"))
+
+
+# Connect to the database.
+conn = pymysql.connect(
+    db='highscores',
+    user='root',
+    passwd='password',
+    host='localhost')
+c = conn.cursor()
+
+def greet_player():
+    if 'first_name' not in form:
+        print("Something went wrong")
+    else:
+        print("<b>Hello " + str(form.getvalue("first_name")) + "</b>")
+        
+'''
+Get information from player: first_name, last_name and score
+'''
+def get_name_score():
+    if 'first_name' and 'last_name' and 'score' not in form:
+        print ("Where are first name, last name, score?")
+    else:
+        first_name = str(form.getvalue('first_name'))
+        last_name = str(form.getvalue('last_name'))
+        score = str(form.getvalue('score'))
+        
+'''        
+I don't know why this won't work. It doesn't give an error so I have no idea what's wrong.
+'''
+def add_player(first_name, last_name):
+# Insert some example data.
+    c.execute("INSERT INTO players VALUES (NULL, ' + first_name + ', ' + last_name + ')")
+    conn.commit()
+    
+'''    
+I don't know if this is working as it gives an error when ever I try to use with add_player or add_score.
+'''
+#def get_id(first_name):
+    #c.execute("SELECT * FROM players WHERE first_name = " ' + first_name + ')
+    #player_id = ([(r[0] for r in c.fetchall()])
+
+'''
+This worked but gave me a weird id "at 0x7f54c1859ea0>".
+'''
+def get_player_id():
+    c.execute("SELECT * FROM players")
+    player_id = str(r[0] for r in c.fetchall())
+    print (player_id)
```

Comment:
```player_id``` will be a list. When you attempt to print a list, it will print it's memory location.